packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -202,6 +212,66 @@ export class DotnetPluginImpl implements PluginImpl {
   }
 
   public async localDebug(ctx: PluginContext): Promise<TeamsFxResult> {
+    if (isConfigUnifyEnabled()) {
+      ctx.envInfo.state.get(PluginInfo.pluginName)?.set(ConfigInfo.indexPath, PathInfo.indexPath);
+    } else {
+      ctx.localSettings?.frontend?.set(LocalSettingsFrontendKeys.TabIndexPath, PathInfo.indexPath);
+    }
+    return ok(undefined);
+  }
+
+  public async postLocalDebug(ctx: PluginContext): Promise<TeamsFxResult> {
+    const appSettingsPath = path.join(ctx.root, PathInfo.appSettingDevelopment);
+    let appSettings = await fs.readFile(appSettingsPath, "utf-8");
```

Comment:
To be robust, check if the file is existing or catch related exceptions.

Reply:
Let's fix it in next PR.